test/trento/users/policy_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule Trento.Users.PolicyTest do
+  use ExUnit.Case
+
+  alias Trento.Abilities.Ability
+  alias Trento.Users.Policy
+  alias Trento.Users.User
+
+  test "should allow read operations if the user has all:all ability" do
+    user = %User{abilities: [%Ability{name: "all", resource: "all"}]}
+
+    Enum.each([:show, :index], fn action ->
+      assert true == Policy.authorize(action, user, User)
+    end)
+  end
+
+  test "should allow read operations if the user has users:all ability" do
+    user = %User{abilities: [%Ability{name: "all", resource: "users"}]}
+
+    Enum.each([:show, :index], fn action ->
+      assert true == Policy.authorize(action, user, User)
+    end)
+  end
+
+  test "should allow write operations if the user has all:all ability" do
+    user = %User{abilities: [%Ability{name: "all", resource: "all"}]}
+
+    Enum.each([:update, :create, :delete], fn action ->
+      assert true == Policy.authorize(action, user, User)
+    end)
+  end
+
+  test "should allow write operations if the user has all:users ability" do
+    user = %User{abilities: [%Ability{name: "all", resource: "all"}]}
+
+    Enum.each([:update, :create, :delete], fn action ->
+      assert true == Policy.authorize(action, user, User)
+    end)
+  end
+
+  test "should disallow other abilities" do
```

Comment:
What is this test for? I would expect some `refute` as the test talks about `disallow`, but checking it, i think we can just simply remove it

Reply:
I refactored the test, for some reason I had the working copy not committed lol